### PR TITLE
Update references from HPCng to Apptainer 

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
Signed-off-by: Pedro Alves Batista <pedro.pesquisapb@gmail.com>

Replaces any reference of HPCng to Apptainer.

* Fixes:
#7

### CODE REVIEW

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Let's use `{orgrepo}` instead of `apptainer/apptainer` like we did with the userdocs.

File: conf.py [hunk 1]
```diff
@@ -120,7 +120,7 @@
 
 html_context = {
     'display_github': True,
-    'github_user': 'hpcng',
+    'github_user': 'apptainer',
     'github_repo': 'singularity-admindocs',
```
Comment (DrDaveD) [inline conf.py:124]: Change this to apptainer-admindocs

**Review by DrDaveD — CHANGES_REQUESTED** [review]
A conflict needs to be resolved

File: conf.py (hunk 1 above)
Comment (pedroalvesbatista) [reply conf.py:124]: Any other Singularity references will do in another PR, as there are some of them remaining.

Comment (pedroalvesbatista) [reply conf.py:124]: In #21 more specifically, as there are more references out there.

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**